### DISCUSSION TITLE
Automation process information

### DISCUSSION THREAD

**Discussion by baywet** [APIs-guru/openapi-directory#1109] [Ideas]
First off, this is a great initiative!

It appears some kind of automation process is in place to refresh any given API description.
However it's not clear where that process lives, where are the logs for it etc.

Right now when somebody wants a new API description to be added, they just ask about that description and provide the link for the most part.

I'm not sure how much additional work is required by the maintainers to make sure the automation is setup.

If those aspects were a bit more transparent, people could know when any description might be out of date because the automation is broken, they might be able to help fix it, etc

**Comment by MikeRalphson** [comment]
Some initial information on the automation is now [here](https://github.com/APIs-guru/openapi-directory/wiki/Development) on the Wiki.
